h2o-py/h2o/plot/_matplotlib.py

```diff
@@ -3,7 +3,8 @@ def get_matplotlib_pyplot(server, raise_if_not_available=False):
     try:
         # noinspection PyUnresolvedReferences
         import matplotlib
-        matplotlib.use("Agg")
+        if server:
+            matplotlib.use("Agg")
```

Comment:
@tomasfryda :  Can you add a comment here that if anything changes, please do an interactive test and outline what that interactive test should be?

Reply:
I added some general comment about testing explain module but I can't say what exactly should the person test. It depends on the changes. In this case any interactive test of plotting functionality would tell you that it's broken. But it's highly dependent on the change and in general changing things around matplotlib is not easy - we support a lot of versions and they behave differently so we should test for multiple versions of matplotlib interactively. And the test shouldn't be just seeing if there is an output - people use the plots to validate their models so we should make sure that the changes don't change the meaning of the plots or if it makes them useless due to some visual artifact, e.g., multiple colors replaced by one, in ice plot predicted value not plotted, wrong labels (i.e. labels don't correspond with the data) etc.

Unfortunately, it's a lot of work testing the plotting functionality but I haven't found a better way than checking everything manually.